src/shared/tpm2-util.h

```diff
@@ -80,7 +80,8 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(Tpm2Context*, tpm2_context_unref);
 typedef struct {
         Tpm2Context *tpm2_context;
         ESYS_TR esys_handle;
-        bool keep;
+
+        bool no_flush;
```

Comment:
as mentioned, please don't do inverted bools

Reply:
ack, changed to 'flush'